concepts/Common/Security Considerations/20 HTML Encoding/20 Encoding Across Components/html.md

```diff
@@ -0,0 +1,27 @@
+Items in collection widgets ([List](/Documentation/ApiReference/UI_Widgets/dxList/), [SelectBox](/Documentation/ApiReference/UI_Widgets/dxSelectBox/), [Toolbar](/Documentation/ApiReference/UI_Widgets/dxToolbar/)) can apply appearance based on data source fields (see the [Default Templates](/Documentation/Guide/Widgets/Common/Templates/#Default_Templates) article). [html](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/items/#html) is one of such fields that specifies item markup. Its values are not encoded. Ensure that they does not contain malicious code. Alternatively, you can use the [text](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/items/#text) field. Unlike **html** values, **text** values are encoded.
```

Comment:
1) `Its values are not encoded.`
Does "Its" refer to the html field in the previous sentence? If so, can we say:

`This field's values are not encoded.`

2) `Ensure that they does not contain malicious code.`
What is "they"? Please specify.

Reply:
1. Yes, we can say so.
2. "They" refer to the field's values.

Reply:
Thanks for the clarification. I've added a suggestion below:

```suggestion
Items in collection widgets ([List](/Documentation/ApiReference/UI_Widgets/dxList/), [SelectBox](/Documentation/ApiReference/UI_Widgets/dxSelectBox/), [Toolbar](/Documentation/ApiReference/UI_Widgets/dxToolbar/)) can apply appearance based on data source fields (see the [Default Templates](/Documentation/Guide/Widgets/Common/Templates/#Default_Templates) article). [html](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/items/#html) is one of such fields that specifies item markup. This field's values are not encoded, so ensure that they do not contain malicious code. Alternatively, you can use the [text](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/items/#text) field. Unlike **html** values, **text** values are encoded.
```